app/app_diagrams_generator.rb

```diff
@@ -116,11 +116,11 @@ def init_system_dependencies
         app_dependencies.each_key do |dep_app|
           dep_app_type = @application_type[dep_app]
 
-          if dep_app_type != key
-            system_dependencies.push dep_app_type unless system_dependencies.include? dep_app_type
-            app_system_dependencies.push dep_app_type unless app_system_dependencies.include? dep_app_type
-            # puts "#{key}:#{app_name} -> #{dep_app_type}:#{dep_app}"
-          end
+          next unless dep_app_type != key
+
+          system_dependencies.push dep_app_type unless system_dependencies.include? dep_app_type
+          app_system_dependencies.push dep_app_type unless app_system_dependencies.include? dep_app_type
+          # puts "#{key}:#{app_name} -> #{dep_app_type}:#{dep_app}"
```

Comment:
It'd be good to kill this line. I'm resisting the urge to comment on the code above it too 😄

Reply:
Oh yeah, I only noticed the indentation change. I've added another commit - what do you think?

Reply:
Excellent 🥇